multicluster/pkg/client/clientset/versioned/clientset.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 Antrea Authors
+// Copyright 2023 Antrea Authors
```

Comment:
Should auto-update the copyright year of generated files? I did not see this for antrea client files. @antoninbas

Reply:
We auto-update the copyright year of generated files any time there is any change in that file: https://github.com/antrea-io/antrea/blob/da6ca963c4aac1a8e0acd3b3afe76e67bc8cb052/hack/update-codegen-dockerized.sh#L112

If the generated file would stay the same, we don't update the year.

Reply:
The reason is the existing license file has one more empty line comparing with old license file in multi-cluster folder. the `reset_year_change` will reset files only when there is only one line changed with the year. I will reset the year change and add the new line only.

Reply:
Done